share/functions/type.fish

```diff
@@ -58,7 +58,7 @@ function type --description 'Print the type of a command'
                             functions $i
                         else
                             set -l func_path (functions --details $i)
-                            if test $func_path != -
+                            if not contains $func_path - stdin
```

Comment:
That definitely should use a `--` guard - `not contains -- $func_path - stdin`.

Reply:
Excellent point. Done.